keylime/cloud_verifier_tornado.py

```diff
@@ -62,10 +63,14 @@ def get_AgentAttestStates():
     "pcr10": "",
     "next_ima_ml_entry": 0,
     "learned_ima_keyrings": {},
+    # filled later from ssl_contexts
     "ssl_context": None,
 }
 
 
+ssl_contexts = {}
```

Comment:
This does not work when Keylime is running on multiple processes. See other comment.